src/codegen/object.rs

```diff
@@ -456,6 +457,11 @@ fn generate_builder(w: &mut dyn Write, env: &Env, analysis: &analysis::object::I
     pub fn build(self) -> {name} {{",
         name = analysis.name,
     )?;
+    writeln!(
+        w,
+        "{}",
+        safety_assertion_mode_to_str(SafetyAssertionMode::InMainThread)
```

Comment:
Don't ask me why this was done like this...

Reply:
What exactly?

Reply:
having to call safety_assertion_mode_to_str instead of just implementing Display or a method in the trait.